.gitignore

```diff
@@ -17,3 +17,4 @@ pytestdebug.log
 ubuntu-xenial-16.04-cloudimg-console.log
 .python-version
 coverage.xml
+pip-wheel-metadata
```

Comment:
I believe that this should probably go to a separate PR. (Personally, though, I'd prefer having it based on the gitignore.io templates as it's easier to keep up with standards this way...)

Reply:
See https://github.com/ansible/molecule/pull/1978.